oaknational/Oak-Web-Application
# Huge image on homepage affecting page performance

## Description
Fix to reduce huge image size on homepage

> [!NOTE]  
> Size in DOM are based on pixel ratio of your browser so might differ 

Reduction

```
243kb - 64kb = 179kb (savings)
```

## How to test

1. Go to https://deploy-preview-3028--oak-web-application.netlify.thenational.academy
2. Check image is of a decent size at all the breakpoints

## Screenshots
Before:
<img width="1122" alt="Screenshot 2024-11-26 at 17 53 36" src="https://github.com/user-attachments/assets/51e0356e-8af1-41da-a16d-899f8856a020">

After:
<img width="1122" alt="Screenshot 2024-11-26 at 17 53 55" src="https://github.com/user-attachments/assets/f2412bff-18ca-4cfd-98d8-ee4d5674f7c0">
